test/testdrive/mzcompose.yml

```diff
@@ -114,7 +114,7 @@ services:
   localstack:
     image: localstack/localstack:0.12.5
     environment:
-    - SERVICES=kinesis,s3
+    - SERVICES=sts,iam,kinesis,s3
```

Comment:
Doesn't much matter but I think we also list what services to use in `guide-testing.md` somewhere, and might be worth updating those too.